### ISSUE TITLE
Validation set accuracies (without human36 data)

### ISSUE THREAD

**Issue by benjiebob** [nkolot/GraphCMR#18]
Hi there,

I just wondered if you could paste validation set accuracies when training from scratch without the human3.6m data...

I ran an experiment over the weekend (50 epochs, batch_size 16, learning rate 3e-4) and get (for example) on up-3d:

Shape Error (NonParam): 309.1791097267133
Shape Error (Param): 311.3052140344197

This is considerably worse than downloaded model file 'data/models/model_checkpoint_h36m_up3d_extra2d.pt' which I assume has also been trained on human36m.

Shape Error (NonParam): 96.80226868738498
Shape Error (Param): 98.1649226233573

Do you think performance without human3.6m data should degrade this much?

**Comment by nkolot** [comment]
Unfortunately I don't have the numbers for that, but the error seems very high. H36M data are crucial to achieve good numbers on H36M  but they should not be crucial to get good results on the "in-the-wild" datasets. How do the results look visually?

**Comment by benjiebob** [comment]
Example results:
<img width="350" alt="Screenshot 2019-07-29 at 11 11 39" src="https://user-images.githubusercontent.com/17785299/62040500-aa113580-b1f1-11e9-8423-bbc8ee488c1f.png">

And loss function which definitely still seems to be decreasing:

<img width="642" alt="Screenshot 2019-07-29 at 11 12 35" src="https://user-images.githubusercontent.com/17785299/62040573-cc0ab800-b1f1-11e9-8bf4-9ec269cbc322.png">

**Comment by nkolot** [comment]
Hmm, if these are training set results, then it's still early in training, because both the shape and the alignment is not very good. Can you let it train for double the number of iterations and report the numbers again?
